src/platforms/ctxlink/WiFi_Server.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+void app_initialize(void);
+void app_task(void);
+//
+void gdb_tcp_server(void);
+bool is_gdb_client_connected(void);
+
+void data_tcp_server(void);
+bool is_uart_client_connected(void);
+void send_uart_data(uint8_t *lpBuffer, uint8_t length);
+
+bool swo_trace_server_active(void);
+void wifi_setup_swo_trace_server(void);
+bool is_swo_trace_client_connected(void);
+void send_swo_trace_data(uint8_t *lpBuffer, uint8_t length);
+
+void wifi_gdb_putchar(unsigned char ch, int flush);
```

Comment:
We believe that `flush` should be `bool` here - you'll need to `#include <stdbool.h>` at the top of the file for that but it should otherwise be good to go.

Reply:
This function is based up the common/stm32/gdb_if.c file, which as:
```void gdb_if_putchar(const char c, const int flush)```